docs/features/G03-anti-anxiety-mechanics.md

```
@@ -1,50 +1,50 @@
 priority: P0
-status: Proposed
+status: Built
 tags: [ux, ranges, non-judgmental, no-streaks, copy, mechanics]
 ---
 
 # Anti-Anxiety Mechanics
 
-**Competitor verdict:** Not a clean gap for the tagline (Napper owns "non-judgmental"), real opportunity in the mechanics — win on concrete UI, not a slogan.
+**Competitor verdict:** Not a clean gap for the tagline (Napper owns "non-judgmental"), real opportunity in the mechanics â€” win on concrete UI, not a slogan.
 
 ## Problem / user need
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)